elisp/init-haskell.el

```diff
@@ -38,6 +38,17 @@
 ;; HaskellModePac
 (use-package haskell-mode
   :mode "\\.hs\\'")
+
+;; Install and configure lsp-haskell
+(use-package lsp-haskell
+  :ensure t
+  :config
+  (setq lsp-haskell-server-path "haskell-language-server")
+  )
+
+(add-hook 'haskell-mode-hook #'lsp)
+(add-hook 'haskell-literate-mode-hook #'lsp)
```

Comment:
Use `:hook` instead, please refer to my other config.

Reply:
sure.